src/modules/layout/components/ModalHeader.js

```diff
@@ -19,12 +19,6 @@ const HorizontalLeft = styled.View`
   align-items: center;
 `
 
-const Title = styled.Text`
- font-size: 30px;
- color: black;
- margin-left: 10px;
-`
-
 const BoxShadow = styled.View`
   background-color: transparent;
```

Comment:
You can use:

```
const BoxShadow = styled.View`
  background-color: transparent;
  position: absolute;
  z-index: 100;
  top: 0;
  left: 0;
  right: 0;
  height: ${props => props.theme.dimensions.modalHeaderHeight};
`
```

That way the ModalHeader is always transparent and does not take space. You can also name this then TransparentHeader maybe. So it is more reusable :)

Reply:
Done. This somehow fixed the bug that the image is not displayed beneath the header. :+1: